domains/arceus.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "blurwind",
+           "email": "96443442+blurwind@users.noreply.github.com",
+           "discord": "1055478076013817936"
+        },
+    
+        "record": {
+            "CNAME": "blurwind.github.io"
```

Comment:
since CNAME record is already redirecting to the is-a.dev domain, can i have a preview of your website?